rskj-core/src/main/java/co/rsk/core/bc/BlockExecutor.java

```diff
@@ -513,7 +672,9 @@ private BlockResult executeSequential(
             logger.trace("track commit");
 
             long gasUsed = txExecutor.getGasUsed();
-            totalGasUsed += gasUsed;
+            long totalGasUsedInBucket = parallelizeTransactionHandler.getGasUsedIn(bucketId.get());
+
+            totalGasUsedInBucket += gasUsed;
```

Comment:
doesn't the gas bucket already include the tx gas used?

Reply:
Indeed! Good catch